src/resources/messages/interfaces.ts

```diff
@@ -0,0 +1,46 @@
+export interface Message {
+  id: string;
+  channel_id: string;
+  recipient: string | ObjectIdentifier;
+  workflow: string;
```

Comment:
I know we return this field from the API but lets remove it from the type declaration so folks use `source`